.github/workflows/deploy-push-master.yaml

```diff
@@ -63,6 +63,7 @@ jobs:
           # Parameters you may likely hardcode here directly:
           deployment_client_id: 5f18d04a-2d2a-41a2-aa15-bd62a149cd14
           deployment_tenant_id: 0d75f6b8-c6b9-4e84-baca-503e08aa7e4a
+          data_set_id: 4972094093286445
```

Comment:
How hard is to replace that with external IDs?

Reply:
1. Can't be done without requiring `datasets:read` which I don't want to add as a required capability (since all API endpoints we use only support "link by ID", we then would need to convert xid to id with API call first).
2. Quite a few of the parameters are not really options that make the action complicated, but rather simple: 8/21 params are credentials + cluster & project, 4/21 are just attributes on the function object. Of the rest, only these are real "user provided settings": `common_folder`, `function_secrets`, `function_deploy_timeout`, `post_deploy_cleanup`, `data_set_id`, and all of these are optional. Conclusion: I do not see any easy way to reduce the number of parameters (can remove then from the workflow file of course and only add to README, but I don't see the point).